app/frontend/src/lib/utils.test.js

```diff
@@ -55,6 +55,13 @@ describe('stringMatchesPostcode', () => {
   });
 });
 
+describe('stringContainsNumber', () => {
+  test('is true when supplied string contains a number otherwise false', () => {
+    expect(stringContainsNumber('alex8')).toBe(true);
+    expect(stringContainsNumber('alex')).toBe(false);
```

Comment:
This could be 2 separate tests I think

Reply:
i had excatly the same thought, yes you are right but at first i couldnt be arsed then i thought its petty. but ill split it will take 2mins